src/simdjson.cpp

```diff
@@ -1,3 +1,7 @@
+// Instead of including simdjson.h, we include the stuff inside so we can get all the
+// implementations and not just builtin
+#define SIMDJSON_CPP
```

Comment:
Is this macro (SIMDJSON_CPP) ever used?

Reply:
Honestly, I don't remember adding it. I'll remove it if It's unused.

Reply:
You have a mysterious comment : "Instead of including simdjson.h, we include the stuff inside so we can get all the implementations and not just builtin."